tests/utils/__init__.py

```diff
@@ -18,6 +20,8 @@
 
 
 def get_vault_version_string():
+    if not find_executable("vault"):
+        raise SkipTest
```

Comment:
Same deal here:

```
$ nosetests -v tests/integration_tests/v1/test_integration.py
Failure: SkipTest () ... SKIP

----------------------------------------------------------------------
Ran 1 test in 0.000s

OK (SKIP=1)
```
versus:

```
def get_vault_version_string():
    if not find_executable("vault"):
        raise SkipTest('CAN NOT FIND NO VAULT')
    command = ['vault', '-version']
```

```
$ nosetests -v tests/integration_tests/v1/test_integration.py
Failure: SkipTest (CAN NOT FIND NO VAULT) ... SKIP: CAN NOT FIND NO VAULT

----------------------------------------------------------------------
Ran 1 test in 0.000s

OK (SKIP=1)
```

To be clear, the motivation in providing a message is mainly so developers have information about why tests are being skipped so they don't end up getting caught unaware by the CI tests run after opening a PR.